packages/client/src/components/DateRangePicker.tsx

```diff
@@ -525,14 +526,19 @@ function DateRangePickerComponent(props: IDateRangePickerProps) {
         </MonthSelectorHeader>
         <MonthButtonsContainer>
           {months.map((month, index) => {
-            const monthDate = new Date(Number(year), index)
+            const monthDate =
+              label === 'From'
```

Comment:
@jamil314  I am afraid label matching may not work with other locales. I can think of one way that you can provide different distinctive id names for start and end month selector components to differentiate.